tools/editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -3279,26 +3283,37 @@ CanvasItemEditor::CanvasItemEditor(EditorNode *p_editor) {
 	handle_len=10;
 	first_update=true;
 
+	Ref<ShortCut> sc_select_mode=ED_SHORTCUT("canvas_item_editor/select_mode",TTR("Select Mode"),KEY_Q);
+	Ref<ShortCut> sc_drag_pivot=ED_SHORTCUT("canvas_item_editor/drag_pivot",TTR("Drag Pivot"),KEY_MASK_SHIFT|KEY_V);
+	Ref<ShortCut> sc_change_pivot=ED_SHORTCUT("canvas_item_editor/change_pivot",TTR("Change Pivot"),KEY_V);
+	Ref<ShortCut> sc_move_mode=ED_SHORTCUT("canvas_item_editor/move_mode",TTR("Move Mode"),KEY_W);
+	Ref<ShortCut> sc_rotate_mode=ED_SHORTCUT("canvas_item_editor/rotate_mode",TTR("Rotate Mode"),KEY_E);
 
 	select_button = memnew( ToolButton );
 	select_button->set_toggle_mode(true);
 	hb->add_child(select_button);
 	select_button->connect("pressed",this,"_tool_select",make_binds(TOOL_SELECT));
 	select_button->set_pressed(true);
-	select_button->set_tooltip(TTR("Select Mode (Q)")+"\n"+keycode_get_string(KEY_MASK_CMD)+TTR("Drag: Rotate")+"\n"+TTR("Alt+Drag: Move")+"\n"+TTR("Press 'v' to Change Pivot, 'Shift+v' to Drag Pivot (while moving).")+"\n"+TTR("Alt+RMB: Depth list selection"));
+	select_button->set_tooltip(TTR("Select Mode")+" ("+sc_select_mode->get_as_text()+")\n"+
+								keycode_get_string(KEY_MASK_CMD)+TTR("Drag: Rotate")+"\n"+
+								TTR("Alt+Drag: Move")+"\n"+
+								TTR("Press ($sc) to Change Pivot, ($sc) to Drag Pivot (while moving).")
+													.replace_first("$sc",sc_change_pivot->get_as_text())
+													.replace_first("$sc",sc_drag_pivot->get_as_text())+"\n"+
+								TTR("Alt+RMB: Depth list selection"));
```

Comment:
IIUC the tooltip won't get updated when the hotkey is changed (but it would be after an editor restart I guess?), is that right? If so, the current behaviour is still OK-ish and we could open an issue to work on the broader matter instead of holding this PR for another month :)